util-scripts/export-cves-to-csv/create-csv.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+
+set -e
+
+if [[ -z "${ROX_ENDPOINT}" ]]; then
+  echo >&2 "ROX_ENDPOINT must be set"
+  exit 1
+fi
+
+if [[ -z "${ROX_API_TOKEN}" ]]; then
+  echo >&2 "ROX_API_TOKEN must be set"
+  exit 1
+fi
+
+if [[ -z "$1" ]]; then
+  echo >&2 "usage: create-csv.sh <output filename>"
+  exit 1
+fi
+
+output_file="$1"
+echo '"Deployment", "Image", "CVE", "CVSS Score", "Summary", "Component", "Version", "Fixed By", "Layer Index", "Layer Instruction"' > "${output_file}"
+
+function curl_central() {
+  curl -sk -H "Authorization: Bearer ${ROX_API_TOKEN}" "https://${ROX_ENDPOINT}/$1"
+}
+
+# Collect all alerts
+cvss=7
+
+res="$(curl_central "v1/alerts?query=Policy:Fixable CVSS >= ${cvss}")"
```

Comment:
URL encode this:
`res="$(curl_central "v1/alerts?query=Policy%3AFixable%20CVSS%20%3E%3D%20${cvss}")"`